README.md

```diff
@@ -71,22 +69,6 @@ An example configuration, used by this image, follows:
 line-length = 88
 target-version = ['py38']
 include = '\.pyi?$'
-exclude = '''
-(
-  /(
-      \.eggs         # exclude a few common directories in the
-    | \.git          # root of the project
-    | \.hg
-    | \.mypy_cache
-    | \.tox
-    | \.venv
-    | _build
-    | buck-out
-    | build
-    | dist
-  )/
-)
-'''
 
 [tool.isort]
 profile = "black"
```

Comment:
I think the `isort` example should use the Plone profile. Now it is compatible with `black`.

https://github.com/PyCQA/isort/pull/1926

But this can be done in another PR.

Reply:
That fix is not in a released version of `isort` yet.

Reply:
Oh, right.